org.eclipse.gef/src/org/eclipse/gef/internal/InternalGEFPlugin.java

```diff
@@ -67,4 +72,45 @@ public void stop(BundleContext context) throws Exception {
 		super.stop(context);
 	}
 
+	/**
+	 * Convenience method for getting the current zoom level of the active device.If
+	 * on MacOS or Linux (x11 window system) or if the device zoom couldn't
+	 * otherwise be determined, this method returns {@code 100} as default value.
+	 */
+	@SuppressWarnings("nls")
```

Comment:
Better add the proper `//$NON-NLS-X$` and remove this, otherwise you'll be possibly silencing valid warnings that may be introduced in future changes.

Reply:
I've updated the PR to include the three things you've pointed out.